shared/chat/conversation/attachment-input/index.js

```diff
@@ -40,7 +40,7 @@ class RenderAttachmentInput extends Component<void, Props, State> {
 
   render() {
     const count = this.props.inputs.length
-    const currentTitle = this.props.inputs[this.state.index].title
+    const currentTitle = this.state.index < count ? this.props.inputs[this.state.index].title : ''
```

Comment:
theoretically this.props.inputs[this.state.index] could point to an empty object, so to check for both cases couldn't we do
```
this.props.inputs[this.state.index] && this.props.inputs[this.state.index].title || ''
```